.github/workflows/conformance-gateway-api.yaml

```diff
@@ -31,8 +39,92 @@ env:
   timeout: 5m
 
 jobs:
+  check_changes:
+    name: Deduce required tests from code changes
+    if: |
+      github.event_name == 'schedule' ||
```

Comment:
`github.event_name == 'schedule'` is checked here but neither workflows are triggered with a schedule.
There are two triggers for the workflows `pull_request` and `push`, both of them already have path filters defined.

The question is what is the expected behavior?

Reply:
I copy-pasted from `.github/workflows/conformance-clustermesh.yaml`. I'm not skilled at GH Actions so it's possible I'm doing something completely wrong. 

The desired behavior is for ConformanceGatewayAPI and ConformanceIngress to be skipped if there are changes only to `test/` directory. However, they get stuck as required jobs as seen in https://github.com/cilium/cilium/pull/25325.